extensions/ql-vscode/src/remote-queries/analyses-results-manager.ts

```diff
@@ -166,4 +168,8 @@ export class AnalysesResultsManager {
   private isAnalysisInMemory(analysis: AnalysisSummary): boolean {
     return this.internalGetAnalysesResults(analysis.downloadLink.queryId).some(x => x.nwo === analysis.nwo);
   }
+
+  private createFileLinkPrefix(nwo: string, sha: string): string {
```

Comment:
Minor: Consider giving this a name that indicates it's a GitHub.com link prefix. In future if we add GHES support that will help us find this code and make it more flexible.